composer.json

```diff
@@ -20,8 +20,6 @@
     },
     "require": {
         "php": "^8.0",
-        "npm-asset/bootstrap": "^5.0.0",
-        "oomphinc/composer-installers-extender": "^2.0.0",
```

Comment:
Add info about this change to changelog.

Reply:
It hasn't been released.